src/main/java/pl/javastart/task/MixContract.java

```diff
@@ -0,0 +1,96 @@
+package pl.javastart.task;
+
+public class MixContract extends PhoneContract {
+    private int smsSent;
+    private int mmsSent;
+    private int secondsCall;
+
+    private int smsLimit;
+    private int mmsLimit;
+    private double callLimit;
+
+    public MixContract(double accountBalance, double smsCost, double mmsCost, double callCost, int smsLimit,
+                       int mmsLimit, double callLimit) {
+        super(accountBalance, smsCost, mmsCost, callCost);
+        this.smsLimit = smsLimit;
+        this.mmsLimit = mmsLimit;
+        this.callLimit = callLimit;
+    }
+
+    public int getSmsLimit() {
+        return smsLimit;
+    }
+
+    public void setSmsLimit(int smsLimit) {
+        this.smsLimit = smsLimit;
+    }
+
+    public int getMmsLimit() {
+        return mmsLimit;
+    }
+
+    public void setMmsLimit(int mmsLimit) {
+        this.mmsLimit = mmsLimit;
+    }
+
+    public double getCallLimit() {
+        return callLimit;
+    }
+
+    public void setCallLimit(double callLimit) {
+        this.callLimit = callLimit;
+    }
+
+    @Override
+    public void sendSms() {
+        if (getSmsLimit() - smsSent >= 0) {
+            smsSent++;
+            System.out.println("SMS wysłany\n");
+        } else if (getAccountBalance() - getSmsCost() > 0) {
+            setAccountBalance(getAccountBalance() - getSmsCost());
+            System.out.println("SMS wysłany\n");
+        } else {
+            System.out.println("Brak środków na koncie. SMS nie został wysłany!\n");
+        }
```

Comment:
Tak naprawdę to masz już w klasie CardContract, może warto zmienić dziedziczenie?